pages.de/common/kubectl-delete.md

```diff
@@ -0,0 +1,32 @@
+# kubectl delete
+
+> Löschen von Kubernetes-Ressourcen.
+> Weitere Informationen: <https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#delete>.
+
+- Löschen eines bestimmten Pods:
+
+`kubectl delete pod {{pod_name}}`
+
+- Löschen eines bestimmten Deployments:
+
+`kubectl delete deployment {{deployment_name}}`
+
+- Löschen eines bestimmten Nodes:
+
+`kubectl delete node {{node_name}}`
+
+- Löschen aller Pods in einem bestimmten Namespaces:
+
+`kubectl delete pods --all --namespace {{namespace}}`
```

Comment:
Please translate the placeholders, like `namespace` or `deployment_name`, for get this easier for non-English speakers.

Reply:
I think we can leave `namespace` untranslated, but `deployment_name` could be something like `Verteilername`.